lib/govuk_elements_form_builder/form_builder.rb

```diff
@@ -44,7 +44,7 @@ def add_error_to_label! html_tag
       message = error_full_message_for attribute
       html_tag.sub!('label', %'label id="error_#{field}"')
       html_tag.sub!('</label',
-        %'<span class="error-message" id="error_message_#{field}">#{message}</span></label')
+        %'<span class="error-message">#{message}</span></label')
```

Comment:
The `error_message` id is being used in the input `aria-describedby` attribute. Recommend we add it back, to be consistent with the GOV.UK elements guide [use of aria-describedby](https://govuk-elements.herokuapp.com/errors/#highlight-errors).